### PR TITLE
Updated the gallery

### LINKED ISSUE #4: Make a new gallery page and add gallery to navbar

### PR DESCRIPTION
**Pull request by StrixROX**
## Closes #4 

1. Replaced the old pics with new ones.
2. Cleaned the code of the gallery page a little. Now it reads all images from `images/gallery` and programmatically adds all images in the page.
(PS: I had to create a naming convention for the images for this. All portrait images should have name like `*-tall.jpg` and all landscape images like `*-wide.jpg`.)
3. Changed the gallery hover animations.
4. Added the gallery link to the navbar on homepage.
image

Demo of gallery:
https://user-images.githubusercontent.com/40167899/103159179-17ad0100-47ec-11eb-8553-9d2a9f32a537.mp4

### CODE REVIEW

File: connect.php [hunk 1]
```diff
@@ -4,7 +4,7 @@
 $host = 'localhost';
 $db   = 'infinitoAdmin';
 $user = 'root';
-$pass = DBPASSWORD;
+$pass = '12345678';
```
Comment (debarghya472) [inline connect.php]: @StrixROX follow the instructions mentioned in updated [contributing.md](https://github.com/Infinito-IIT-Patna/Infinito2020/blob/master/CONTRIBUTING.md) to setup the database.

**Review by debarghya472 — CHANGES_REQUESTED** [review]
Change the db setup accordingly and also share screenshots or videoclips to demonstrate the working of gallery page.

File: connect.php (hunk 1 above)
Comment (StrixROX) [reply connect.php]: Sorry about that! Will resolve and send pics/vids asap

Comment (StrixROX) [reply connect.php]: Added the gallery link:
image

Demo of gallery:
https://user-images.githubusercontent.com/40167899/103159179-17ad0100-47ec-11eb-8553-9d2a9f32a537.mp4

**Review by debarghya472 — APPROVED** [review]
Great Job!!

**Pull request merged**